src/main/java/org/opensearch/ad/AnomalyDetectorJobRunner.java

```diff
@@ -502,7 +504,12 @@ private void indexAnomalyResult(
             // For a HCAD detector, we don't need to save on the detector level.
             // We return 0 or Double.NaN rcf score if there is no error.
             if ((response.getAnomalyScore() <= 0 || Double.isNaN(response.getAnomalyScore())) && response.getError() == null) {
-                updateRealtimeTask(response, detectorId);
+                threadPool
+                    .schedule(
+                        () -> { updateRealtimeTask(response, detectorId); },
+                        new TimeValue(60, TimeUnit.SECONDS),
```

Comment:
Could this be set to an even shorter amount of time? Or in your experience is 60s a good amount of time to ensure trained models?

Reply:
60s is a good amount of time to ensure trained models. But it can be longer due to various reasons.

Reply:
If it's longer, the only impact is that the cold start will take one more interval, correct? (assuming it is fully trained by the next interval)

Reply:
yes

Reply:
By ensure trained model? what does this exactly include? I understand we need one interval first to trigger cold start, these 60 seconds are to get back the model from RCF from the data from the the one interval we needed? make sure its on the checkpoint index?

Reply:
we also update our real-time cache every-time we get a non-null, no error or non 0 anomaly score right?

Reply:
60s is to ensure cold start and writing back to checkpoint index. 

For our real-time cache, we don't update if the current value is the same as the cached values.